src/tapcell/src/tapcell.tcl

```diff
@@ -477,6 +477,10 @@ proc tapcell { args } {
     foreach blockage [$block getInsts] {
         set inst_master [$blockage getMaster]
         if { [string match [$inst_master getType] "BLOCK"] } {
+            if { [string match [$blockage getPlacementStatus] "NONE"] } {
+                puts "\[ERROR\] Macro [$blockage getName] is not placed"
+                exit 1
```

Comment:
Errors are not fatal per https://docs.google.com/document/d/13yQg1DqLESm_odyIrMDzJx1sScug8Bii9A8uTgYoufo/edit?pli=1

I think it is ok to just ignore such instances (ie continue) with the error msg.

Reply:
I've update tapcell code to just ignore these instances